app/shared/requiresSignin.ts

```diff
@@ -1,9 +1,19 @@
 import * as pathLib from "path";
+import parse from "url-parse";
 
 // To avoid security vulnerabilities do not add public paths that do not end in a slash
-const publicPaths = ["/api/contact-us/", "/help-centre/", "/api/known-issues/"];
+const publicPaths = [
+  "/api/contact-us/",
+  "/help-centre/",
+  "/api/known-issues/",
+  "/cancel-reminders/",
+  "/api/reminders/cancel/"
+];
```

Comment:
While you're here you would mind ordering this alphabetically? Thanks!

```suggestion
const publicPaths = [
  "/api/contact-us/",
  "/api/known-issues/",
  "/api/reminders/cancel/",
  "/cancel-reminders/",
  "/help-centre/"
];
```

Reply:
👍